src/BUILD

```diff
@@ -0,0 +1,20 @@
+cc_library(
+    name = "atlas_core",
+    srcs = [
+        "atlas.cc",
+        "atlas.h",
```

Comment:
Move this to 'hdrs' if this is the public interface.

Reply:
done